pipelines/frontend-pipeline.yml

```diff
@@ -18,21 +18,11 @@ stages:
     jobs:
       - job: BuildJob
         steps:
-          - task: Bash@3
-            displayName: Generate frontend configurations.
-            inputs:
-              workingDirectory: './frontend'
-              targetType: 'inline'
-              script: |
-                chmod +x generate-fe-configs.sh
-                ./generate-fe-configs.sh -u $(businessUnit) -a $(appName) -e $(environment)
```

Comment:
Removes the creation of the `.env` files.  Since variable groups essentially create environment variables they just need to be set in the variable group and everything else flows through correctly.